CHANGELOG

```diff
@@ -1,3 +1,31 @@
+snowplow-fractribution 0.2.0 (2023-0X-XX)
```

Comment:
Just a kind reminder not to forget to update the dates here when you get to the release.